packages/php/blueprint/src/ExportSchema.php

```diff
@@ -101,4 +102,12 @@ function ( $exporter ) {
 
 		return $schema;
 	}
+
+	public function onBeforeExport($step_name, $callback) {
```

Comment:
Should we also fix lint issues for code in the blueprint package?

Reply:
Lint isn't setup for the package yet. I'll create a new issue for it.